frontend/src/components/atoms/Button/Button.model.ts

```diff
@@ -0,0 +1,3 @@
+export interface IButton extends React.HTMLAttributes<HTMLButtonElement> {
+  type?: 'button' | 'submit' | 'reset';
```

Comment:
Not sure that we need this props `type`. This is default property of HTML button